addons/hr_work_entry_contract/models/hr_contract.py

```diff
@@ -277,3 +279,24 @@ def _recompute_work_entries(self, date_from, date_to):
     def _get_fields_that_recompute_we(self):
         # Returns the fields that should recompute the work entries
         return ['resource_calendar_id']
+
+    @api.model
+    def _cron_generate_missing_work_entries(self):
+        today = fields.Date.today()
+        start = today + relativedelta(day=1)
+        stop = today + relativedelta(day=31)
+        contracts = self.env['hr.employee'].search([])._get_all_contracts(
```

Comment:
```suggestion
        contracts = self.env['hr.employee']._get_all_contracts(
```
I may be missing sthg, but when looking at `_get_all_contracts()`, it's an api.model method doing a search on the model anyway.  Then the first search is unnecessary :thinking:.